Cannot create scaffold other than default example (Books)

<!--- Provide a general summary of the issue in the Title above -->

## This is:
<!-- Select one -->
- Bug

<!--- If you report bug, please follow template below -->
<!--- It will allow us to better understand problem and help you faster -->

## Specifications

  - Version: 5.2.0
  - OS: Ubuntu 20.04
  - Browser: N/A

## Expected Behavior
<!--- Tell us what should happen -->
CLI scaffold should create a custom GraphQL type

## Actual Behavior
<!--- Tell us what happens instead of the expected behavior -->
Scaffold crashes with exception

```bash
yarn webiny scaffold
? Pick a template to scaffold GraphQL API package
? Enter the name of the initial data model Session
? Enter the package location packages/sessions/api
? Enter the package name @sessions/api
✔ Service files created in /home/dev/webiny-test/packages/sessions/api.
✔ Added @sessions/api to api package.json.
✔ Workspace packages/sessions/api added in root package.json.
✔ Package name set into package.json.
✔ Update package tsconfig extends path.
✔ Update package tsconfig.build extends path.
✔ Updated base tsconfig compilerOptions.paths.
✔ Dependencies installed.
WError: Unable to install dependencies. Try running "yarn" in project root manually.
    at Object.generate (/home/dev/webiny-test/node_modules/@webiny/cli-plugin-scaffold-graphql-service/index.js:359:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async scaffold (/home/dev/webiny-test/node_modules/@webiny/cli-plugin-scaffold/scaffold.js:54:5) {
  code: 'Command failed with exit code 1: yarn build\n' +
    'Successfully compiled 14 files with Babel (1378ms).\n' +
    `src/es.ts(1,10): error TS2305: Module '"./types"' has no exported member 'ListBooksWhere'.`,
  data: null
}
```

## Steps to Reproduce the Problem
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
1. Run ```yarn webiny scaffold```
2. Select ```GraphQL API package```
3. Enter anything other than Books for the initial data model
4. Accept defaults for further prompts

## Detailed Description
<!--- Provide a detailed description of the change or addition you are proposing -->
As above

## Possible Solution
<!--- Not obligatory, but suggest a fix/reason for the bug, -->
This is probably a problem with the Books example being hard-coded? I haven't checked the code so that's my first guess.

--- a/packages/cli-plugin-scaffold-graphql-service/template/src/es.ts
+++ b/packages/cli-plugin-scaffold-graphql-service/template/src/es.ts
@@ -1,9 +1,9 @@
-import { ListBooksWhere } from "./types";
+import { ListTargetsWhere } from "./types";
 
 /**
  * A basic filtering functionality for Elasticsearch.
  */
-export const createElasticsearchQuery = (where?: ListBooksWhere) => {
+export const createElasticsearchQuery = (where?: ListTargetsWhere) => {
     if (!where || Object.keys(where).length === 0) {
         return undefined;
     }

